### ISSUE TITLE
Have any idea to support ionic 4?

### ISSUE THREAD

**Issue by netsesame2** [Riron/ionic-img-viewer#128]
These days, ionic4 beta has been published. Image-view can't work anymore.

Hope to be updated soon, it will be great!

**Comment by stewones** [comment]
+1

**Comment by wlclass** [comment]
+1

**Comment by aleygues** [comment]
+1

**Comment by chrisvor** [comment]
Now that Ionic 4 is final and released, this would be awesome! +1000 :)

**Comment by aleygues** [comment]
+1

**Comment by selvam326** [comment]
It is support with ionic 4 beta? plz help.

**Comment by JerryBels** [comment]
Ionic 4 is out of beta :) Seems like a good time to implement this !

**Comment by AndrWeisR** [comment]
Yeah, I'd like this too. I'm trying to port an app from Ionic 3 to 4, and the image viewer is broken.

**Comment by SimonGolms** [comment]
I was also looking for a solution for Ionic 4 and in the end I published my first npm package. So if you are still looking for a solution, you can give [ngx-ionic-image-viewer](https://github.com/SimonGolms/ngx-ionic-image-viewer) a try. 

_Sorry for the self-promotion... in the end I just want to help the community. If it should be against any rules, feel free to delete my post_

**Comment by andrey3diq** [comment]
Thanks @SimonGolms, your plugin is working perfectly!

**Comment by NgYueHong** [comment]
@SimonGolms It would be cool if your plugin has the animation effect like ionic-img-viewer.
